src/view/styles/Microbit.css

```diff
@@ -14,16 +17,24 @@ svg.sim.grayscale {
 }
 .sim-button {
     pointer-events: none;
+    fill: "rgb(17, 17, 17)";
+}
+.sim-button:active {
+    fill: orange;
 }
 .sim-board,
 .sim-display,
 sim-button {
     fill: #111;
 }
 .sim-button-outer:hover {
-    stroke: grey;
-    stroke-width: 3px;
+    stroke: orange !important;

```

Comment:
Do you think we can avoid using `!important`?

Reply:
Yes! Removing it. Shouldn't be used when not necessary. Good catch